webpack.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = (env) => {
           exclude: /node_modules/,
           loader: 'babel-loader',
           options: {
-            presets: ['es2015', 'react'],
+            presets: ['es2015', 'react', 'stage-3'],
```

Comment:
What do you use `stage-3` for? I tried to read some basics about what it is, but I cannot find what new features you use.